### DISCUSSION TITLE
How to Handle Lost Crypto Sent Over Wrong Network

### DISCUSSION THREAD

**Discussion by AdamMattas** [eprbell/rp2#130] [Q&A] [answered]
I am wondering how to handle an intra transaction that never arrived at its intended destination.

**Comment by haavardf** [comment]
I had a quick look at the source code, and I can't find a transaction_type for that. Probably something that needs to be implemented, but I would wait for someone that knows the code to confirm this.

What I would have tried, if it was me, was to mark the transaction's recived as 0, and set the fee to the value that you sent. If that works, make a note what really happened if audited. I have just been looking at this project recently, so I might be completely wrong.

**Reply by AdamMattas** [reply]
That's an interesting suggestion to account for the transaction. If there isn't a built-in procedure for this scenario I will most likely use your method. Thanks for the response.

**Comment by eprbell** [comment]
> I am wondering how to handle an intra transaction that never arrived at its intended destination.

Can you clarify what you mean by "never arrived at its intended destination":
* it was sent to different address than the intended one or 
* it was never sent (e.g. fees are higher than the sender wants to pay) or
* something else.

**Reply by AdamMattas** [reply]
I think I either used an incorrect wallet address or a wallet address on the wrong type of network. I vaguely remember, in 2021, trying to send MATIC to Coinbase and later realizing they weren't using the Polygon network yet.

| Timestamp            | Asset | From Exchange | From Holder | To Exchange | To Holder   | Spot Price | Crypto Sent | Crypto Received | Unique ID                             | Notes                     |
|----------------------|-------|---------------|-------------|-------------|-------------|------------|-------------|-----------------|---------------------------------------|---------------------------|
| 2021-11-12 06:45:00+0000 | MATIC | Polygon       | Adam Mattas | Coinbase    | Adam Mattas | $1.69      | 13.00000000 | 0.00000000      | e53904cf-6dc7-4d14-aa67-4eb07c5b4901 | Never arrived. Wrong network? |

The problem may be more complicated. The following year, Coinbase shows that I received the same amount of MATIC. I haven't been able to account for its origin, and I'm still trying to determine if it's from the same transaction or a later transaction that coincidentally is for the same amount.

**Reply by eprbell** [reply]
> I think I either used an incorrect wallet address or a wallet address on the wrong type of network. I vaguely remember, in 2021, trying to send MATIC to Coinbase and later realizing they weren't using the Polygon network yet.
> Timestamp 	Asset 	From Exchange 	From Holder 	To Exchange 	To Holder 	Spot Price 	Crypto Sent 	Crypto Received 	Unique ID 	Notes
> 2021-11-12 06:45:00+0000 	MATIC 	Polygon 	Adam Mattas 	Coinbase 	Adam Mattas 	$1.69 	13.00000000 	0.00000000 	e53904cf-6dc7-4d14-aa67-4eb07c5b4901 	Never arrived. Wrong network?
> 
> The problem may be more complicated. The following year, Coinbase shows that I received the same amount of MATIC. I haven't been able to account for its origin, and I'm still trying to determine if it's from the same transaction or a later transaction that coincidentally is for the same amount.

I don't know enough about Matic and Polygon to answer definitively. What happened to the coins you sent? Did you lose them or recover them?

**Reply by AdamMattas** [reply]
I'm really just trying to figure out what is the best way to account for the loss of crypto in general. @haavardf used the term "unrecoverable" to describe when crypto is lost for any reason. Is there a method for either scenario where the crypto is permanently lost or is somehow able to be recovered years later?

**Reply by eprbell** [reply, marked as answer]
Hm, it sounds like we may need LOSS as a new type of OUT transaction...

**Reply by AdamMattas** [reply]
It must be difficult to come up with categories for every possible scenario. Thanks for all your work on this software. It has been of great value to me.

**Reply by eprbell** [reply]
Thanks! I'm glad you found it useful. Let me think a bit more about this: I'll update this thread when I have more news.

**Comment by haavardf** [comment]
You can defenitly loose crypto by mistake, I am paying for an online crypto tax solution, and they use the term 'unrecovrable'. It could be sent to the wrong address, scam or even loss of private keys. I guess it will just be an forever non taxable event, but your balance should go to zero.
